roles/nat64_appliance/molecule/default/converge.yml

```diff
@@ -108,6 +129,23 @@
         name: br-mol
         uri: 'qemu:///system'
 
+    # TODO(hjensas): With the zone set in libvirt XML this bridge "should"
+    # already be in this zone. But logs indicate that it is not.
+    # See: https://libvirt.org/formatnetwork.html
+    - name: Make sure br-mol bridge is in the libvirt firewalld zone
+      become: true
+      ansible.posix.firewalld:
+        zone: libvirt
+        interface: br-mol
+        state: enabled
+        permanent: true
```

Comment:
would setting `immediate: true` here (https://docs.ansible.com/ansible/latest/collections/ansible/posix/firewalld_module.html#parameter-immediate) save as the restart of the service? No idea if it would save much time, or if it's actually a good idea, just asking